### PR TITLE
Changed options menu title (depends on Jabref/JabRef#9768)

### PR DESCRIPTION
**Pull request by calixtus**

### CODE REVIEW

File: en/setup/citationkeypatterns.md
Comment (calixtus) [inline en/setup/citationkeypatterns.md:155]: @ThiloteE do you have the time to check on this doc and update the image here? My KDE snipping tool wont allow me to take screenshots of open menus...

Comment (ThiloteE) [reply en/setup/citationkeypatterns.md:155]: can do it, but if i change it via gitbook, would that not create a conflict with this pull-request?

Comment (calixtus) [reply en/setup/citationkeypatterns.md:155]: Probably yes, so let's wait until this is merged

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**
